capacity4more/behat/features/homepage.feature

```diff
@@ -0,0 +1,62 @@
+Feature: Test homepage activity stream
+  In order to see recent content
+  As a group member and non-member and anonymous user
+  I need to be able to see an activity stream of recent operations
+
+  @javascript
+  Scenario: Change one group access to restricted.
```

Comment:
In this case we probably will have to change many other tests. So I think we can change access,  check what we want and change it back.